Libraries/LibWeb/DOM/Document.cpp

```diff
@@ -740,7 +740,9 @@ WebIDL::ExceptionOr<Document*> Document::open(Optional<String> const&, Optional<
     if (m_active_parser_was_aborted)
```

Comment:
Commit message: it's -> its

Also, doesn't seem auto-wrapped at 72 characters?